ui-tests/tests/src/tests/TransferDashboardPage.test.ts

```diff
@@ -0,0 +1,100 @@
+import { Selector } from 'testcafe';
+import { config } from '../config';
+import { LoginPage } from '../page-objects/pages/LoginPage';
+import { ErrorRow, Transfer, TransferDashboardPage, TransferRow, Error } from '../page-objects/pages/TransferDashboardPage';
+import fs from 'fs';
+import xlsx from 'xlsx';
+
+fixture `Transfer Dashboard Feature`
+  .page`${config.pm4mlEndpoint}/transfer`
+  .beforeEach(async (t) => {
+    // Login if not logged in
+    if(Selector('.login-pf-header')) {
+      await t.typeText(LoginPage.usernameField, config.credentials.test.username)
+      await t.typeText(LoginPage.passwordField, config.credentials.test.password)
+      await t.click(LoginPage.loginButton)
+    }
+  });
+
+test.meta({
+  ID: '',
+  STORY: '',
+  description: '',
+})('Can download excel spreadsheet of transfers in basic search', async (t) => {
+  const expectedFilePath = `${process.env.HOME}` + '/Downloads/' + `Payment_Manager_Transfers_${new Date().toDateString()}.xlsx`;
```

Comment:
can /Downloads/ be part of `` instead of as a separate '' @kleyow

Reply:
done.

Reply:
done.